persistent-mysql-2.6.2 haddock failure

In Stackage Nightly build:

```
Preprocessing library for persistent-mysql-2.6.2..
Running Haddock on library for persistent-mysql-2.6.2..
Warning: The documentation for the following packages are not installed. No
links will be generated to these packages: array-0.5.2.0, base-4.10.0.0,
binary-0.8.5.1, bytestring-0.10.8.2, containers-0.5.10.2, deepseq-1.4.3.0,
directory-1.3.0.2, fail-4.9.0.0, filepath-1.4.1.2, ghc-boot-th-8.2.1,
ghc-prim-0.5.1.0, integer-gmp-1.0.1.0, pretty-1.1.3.3, process-1.6.1.0,
template-haskell-2.12.0.0, time-1.8.0.2, transformers-0.5.2.0, unix-2.7.2.2

Database/Persist/MySQL.hs:1053:3: error:
    parse error on input ‘-- ^ Copy the field directly from the record.’
     |
1053 |   -- ^ Copy the field directly from the record.
     |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Haddock coverage:
```

--- a/persistent-mysql/ChangeLog.md
+++ b/persistent-mysql/ChangeLog.md
@@ -1,3 +1,7 @@
+## 2.6.2.1
+
+* Fix haddock documentation [#725](https://github.com/yesodweb/persistent/pull/725)
+
 ## 2.6.2
 
 * Extend the `SomeField` type to allow `insertManyOnDuplicateKeyUpdate` to conditionally copy values.

--- a/persistent-mysql/Database/Persist/MySQL.hs
+++ b/persistent-mysql/Database/Persist/MySQL.hs
@@ -1048,12 +1048,13 @@ insertOnDuplicateKeyUpdate record =
 -- | This type is used to determine how to update rows using MySQL's
 -- @INSERT ON DUPLICATE KEY UPDATE@ functionality, exposed via
 -- 'insertManyOnDuplicateKeyUpdate' in the library.
+--
+-- @since 2.6.2
 data SomeField record where
+  -- | Copy the field directly from the record.
   SomeField :: EntityField record typ -> SomeField record
-  -- ^ Copy the field directly from the record.
+  -- | Only copy the field if it is not equal to the provided value.
   CopyUnlessEq :: PersistField typ => EntityField record typ -> typ -> SomeField record
-  -- ^ Only copy the field if it is not equal to the provided value.
-  -- @since 2.6.2
 
 -- | Copy the field into the database only if the value in the
 -- corresponding record is non-@NULL@.

--- a/persistent-mysql/persistent-mysql.cabal
+++ b/persistent-mysql/persistent-mysql.cabal
@@ -1,5 +1,5 @@
 name:            persistent-mysql
-version:         2.6.2
+version:         2.6.2.1
 license:         MIT
 license-file:    LICENSE
 author:          Felipe Lessa <felipe.lessa@gmail.com>, Michael Snoyman

